src/snowflake/cli/api/project/definition.py

```diff
@@ -102,16 +102,6 @@ def generate_local_override_yml(
     return project.update_from_dict(local)
 
 
-def default_app_package(project_name: str):
-    user = clean_identifier(get_env_username() or DEFAULT_USERNAME)
-    return append_to_identifier(to_identifier(project_name), f"_pkg_{user}")
```

Comment:
merged into the body of `NativeAppProjectModel.package_name` since this was the only usage of this function (and we needed to separate the addition of the `_pkg` from the addition of the username)